app/views/subscriptions/_form.html.haml

```diff
@@ -1,10 +1,10 @@
 = form_for @subscription, :builder => DefaultFormBuilder do |f|
   = f.hidden_field :resource_type, :value => @type.camelcase
-  = f.text_field :resource_id, :placeholder => "Choose a #{@type.humanize.downcase}", :required => true, :label => @type.humanize
-  = f.text_field :taxon_id, :placeholder => t(:choose_a_taxon)
+  = f.text_field :resource_id, :placeholder => t("choose_a_#{@type.humanize.downcase}".to_sym), :required => true, :label => t(@type.to_sym)
```

Comment:
Nothing broken here, but when using `t` with dynamic values like this it's usually good practice to set a `default`, otherwise you might get some ugly results if the key hasn't been translated into the user's locale.